examples/workflows/README.md

```diff
@@ -0,0 +1,53 @@
+
+The promotional workflow uses 5 GitHub Actions workflows: 
+- `update-dev.yml` - this is the *Update Dev Dependencies* workflow, this workflow can be run periodically or tiggered by a release in an upstream infrastructure module repo
+- `update-stage.yml` - this is the *Update Stage Dependencies* workflow
+- `update-prod.yml` - this is the *Update Prod Dependencies* workflow
```

Comment:
Other than dev, stage, and prod as the targets, are there any differences between these?

Reply:
The events they listen for and the events that they send